desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -1794,7 +1794,13 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
       });
 
       self.interpreters = ko.pureComputed(function() {
-        return window.getLastKnownConfig().app_config.editor.interpreters;
+        var interpreters_list = []
+        window.getLastKnownConfig().app_config.editor.interpreters.filter(function(interpreter) {
+          if (interpreter['is_sql'] === true) {
+            interpreters_list.push(interpreter);
+          }
+        });
+        return interpreters_list;
```

Comment:
Simplify to:

return window.[...].interpreters.filter(function (interpreter) { return interpreter.is_sql });

Reply:
Done